Assets/Scripts/Card/SimpleCard.cs

```diff
@@ -2,45 +2,73 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
-
 public class SimpleCard : MonoBehaviour
 {
     GameObject cardFace;
-    Action<SimpleCard> clickAction;
     public Guid cardGuid;
     public bool isSelected { get; private set; } = false;
+    public bool isPlacedOnBattlefield { get; private set; } = false;
```

Comment:
Isn't used anywhere. For now it is just variable that will be required in further.